recipes/libcurl/all/conanfile.py

```diff
@@ -313,6 +312,13 @@ def _get_configure_command_args(self):
         if self._has_zstd_option:
             params.append("--with-zstd={}".format(yes_no(self.options.with_zstd)))
 
+        # Support for metalink was removed in version 7.78.0 https://github.com/curl/curl/pull/7176
+        if tools.Version(self.version) >= "7.78.0":
+            if self.options.with_libmetalink:
+                raise ConanInvalidConfiguration("Support for metalink was removed in libcurl version 7.78.0")
```

Comment:
typically we have this in configure and delete the option... no need to raise

```suggestion
        if tools.Version(self.version) >= "7.78.0":
            if self.options.with_libmetalink:
                raise ConanInvalidConfiguration("Support for metalink was removed in libcurl version 7.78.0")
```

Reply:
@prince-chrismc Suggestion is the same as the original. Mistake?